src/Microsoft.Identity.Client/TokenCacheExtensions.cs

```diff
@@ -43,48 +44,53 @@ namespace Microsoft.Identity.Client
     /// format, and the new unified cache format, common to ADAL.NET, MSAL.NET, and other libraries on the same platform (MSAL.objc, on iOS)</remarks>
     public static class TokenCacheExtensions
     {
+        private const string AccessTokenKey = "access_tokens";
```

Comment:
When reading the code below... IMO, it should have the extra s (AccessTokensKey) , same for the other keys into the dictionary.